apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -35,6 +32,10 @@ const styles = {
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white
   },
+  image: {
+    width: 485,
+    height: 260
+  },
```

Comment:
this seem redundant, but was necessary to constrain the image size on /yourschool because pegasus serves the image differently than dashboard